src/templates/index.html

```diff
@@ -63,14 +63,16 @@
 
                     <div class="row">
                         <div class="input-field col s6">
-                            <input placeholder="For eg: 1" id="first_name" type="number" name="range_low" class="validate">
+                            <input placeholder="For eg: 1" id="first_name" type="number" name="range_low"
+                                class="validate">
```

Comment:
Do we need to a new line here?
_Trailing content of previous line._

Reply:
I think my editor did it.